www/components/card/card.js

```diff
@@ -43,11 +43,13 @@ class Card extends LitElement {
 
   render() {
     return html`
-      ${this.renderImage()}
-      ${this.renderTitle()}
-      <slot name="cardcontent"></slot>
+      <div class="gwd-card">
```

Comment:
Unfortunately this wont work for `yarn serve` because this moves the `<slot>` away from being a direct descendent of the `:host` 😞